qemu/tests/cgroup.py

```diff
@@ -1136,6 +1137,9 @@ def _generate_verification(cpusets, cpus):
         if vm_cpus != params.get("smp", 0):
             logging.info("Expected VM reload.")
             params['smp'] = vm_cpus
+            if arch.ARCH in ('ppc64', 'ppc64le'):
```

Comment:
Guest's vCPU can be set with vcpu_maxcpus AND/OR smp, so `vm_cpus != params.get("smp", 0):` should change to `vm_cpus != (params.get_numeric("smp") or params.get_numeric("vcpu_maxcpus"):`. BTW, `params.get("smp", 0)` is a string.

Reply:
Thanks for pointing it out, it's not my modification this time, I think it makes sense so I will help to update it .

Reply:
Updated